### PR TITLE
[NO MERGE] Update the tokens of all repos in one go

### PR DESCRIPTION
**Pull request by isuruf**
This uses @jjhelmus 's script. It would only change the tokens and would not rerender. It also skips a repo if the repo has already been added.

This would need to be merged to a new branch

cc @conda-forge/core

### CODE REVIEW

File: .CI/update_token.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+from __future__ import print_function
```
Comment (jakirkham) [inline .CI/update_token.py]: Would it be possible to use the same functionality from `conda-smithy` or are there some issues with that approach?

**Comment by jakirkham** [comment]
Really cool idea, @isuruf! Wondering to what degree we are duplicating existing functionality, but otherwise am interested in this approach. We probably could change the Travis CI image in this branch to a `sudo`-free Linux one for faster startup and conversion.

**Comment by jakirkham** [comment]
Side note: If you add `[skip lint]` in commit messages, that will silence the linter. Useful for cases like these were recipes are not being added.

File: .CI/update_token.py (hunk 1 above)
Comment (isuruf) [reply .CI/update_token.py]: I copied over @jjhelmus's code, but why not? I'll update to use conda-smithy

**Comment by isuruf** [comment]
I think we'd have to restart the travis-ci run 1 or 2 times, but that's fine. This updating of tokens is not supposed to happen everyday.

File: .CI/update_token.py (hunk 1 above)
Comment (jakirkham) [reply .CI/update_token.py]: Makes sense. Ok. If you do run into issues using `conda-smithy`, it would be good to know what they are so we can fix them.

Comment (isuruf) [reply .CI/update_token.py]: Done

**Comment by isuruf** [comment]
Seems travis-ci is backlogged these days, but `.travis.yml` is where all the tokens are.

**Comment by isuruf** [comment]
I can merge this to a new branch and work on this in the weekend  (when there is low travis-ci traffic) if there's no objections.

**Comment by ocefpaf** [comment]
Go for it. I will merge https://github.com/conda-forge/staged-recipes/pull/3790 today and that is it :wink:

**Comment by isuruf** [comment]
Thanks

**Comment by jakirkham** [comment]
Would it make sense to add the ssh key to CircleCI in the `token` branch to, @isuruf, or would it be worth creating a new branch for that workflow?

**Pull request merged**